fuel-core/src/executor.rs

```diff
@@ -1516,6 +1507,35 @@ mod tests {
             assert_eq!(validated_block.transactions(), produced_txs);
         }
 
+        #[tokio::test]
+        async fn execute_cb_command() {
```

Comment:
It looks like the test is just executing the script containing the CB opcode and expecting that the execution doesn't error. Would it be prudent to test that we get the expected return value? Perhaps a test that asserts an error when it's expected?

Reply:
Ah yeah I was thinking of adding that as well, it would require a more advanced script though.